test/nightly/global.resource.test.ts

```diff
@@ -46,3 +56,29 @@ async function deleteResourceGroups(): Promise<void> {
         }
     }));
 }
+
+async function createAccount(accountType: RegExp): Promise<void> {
+    // Cosmos DB account must have lower case name
+    const accountName: string = randomUtils.getRandomHexString(12).toLowerCase();
+    const resourceGroupName: string = randomUtils.getRandomHexString(12);
```

Comment:
@wwlorey discovered today that there were quite a few Cosmos DB accounts that weren't being cleaned up properly, and I wonder if it's related to this PR. You should always add the following line _right after_ generating a resource group name:
```
resourceGroupsToDelete.push(resourceGroupName);
```

NOTE: Once you add it here, you can remove that line from `suiteSetup` in the other three files